public/js/views/transaction/complete-payment.jsx

```diff
@@ -27,14 +27,28 @@ export default class CompletePayment extends Component {
   handleClick = (e) => {
     e.preventDefault();
     var win = this.props.win;
-    if (win.parent !== window) {
+    var _parent = win.parent;
+
+    // TODO: wrap this so dead code elimination removes it
+    // it's just to make the OK button work when served by
+    // webpack-dev-server.
+    try {
+      if (_parent.location.hostname === 'pay.webpack') {
+        _parent = _parent.parent;
```

Comment:
it might be nice to add a log message here saying that `_parent` was pushed up a level